rllib/agents/qmix/qmix_policy.py

```diff
@@ -125,7 +147,7 @@ def forward(self, rewards, actions, terminated, mask, obs, next_obs,
         return loss, mask, masked_td_error, chosen_action_qvals, targets
 
 
-class QMixTorchPolicy(Policy):
+class QMixTorchPolicy(TorchPolicy):
```

Comment:
We should probably call `TorchPolicy.__init__` if extending TorchPolicy. I'm not sure if extending it is necessary though, since QMix doesn't really use the common facilities provided by TorchPolicy.